tls/s2n_security_policies.c

```diff
@@ -839,6 +863,53 @@ const struct s2n_security_policy security_policy_pq_20231215 = {
     },
 };
 
+/* Same as security_policy_aws_crt_sdk_tls_12_06_23 but with ML-KEM Support */
+const struct s2n_security_policy security_policy_aws_crt_sdk_tls_12_06_23_pq = {
+    .minimum_protocol_version = S2N_TLS12,
+    .cipher_preferences = &cipher_preferences_aws_crt_sdk_default,
+    .kem_preferences = &kem_preferences_pq_tls_1_3_mixed_2024_10,
+    .signature_preferences = &s2n_signature_preferences_20200207,
+    .ecc_preferences = &s2n_ecc_preferences_20230623,
+};
+
+/* Same as security_policy_pq_tls_1_2_2023_10_07, but with TLS 1.2 Kyber removed, and added ML-KEM support */
+const struct s2n_security_policy security_policy_pq_tls_1_2_2024_10_07 = {
+    .minimum_protocol_version = S2N_TLS12,
+    .cipher_preferences = &elb_security_policy_tls13_1_2_Ext2_2021_06,
+    .kem_preferences = &kem_preferences_pq_tls_1_3_mixed_2024_10,
+    .signature_preferences = &s2n_signature_preferences_20200207,
+    .ecc_preferences = &s2n_ecc_preferences_20200310,
+};
+
+/* Same as security_policy_pq_tls_1_2_2024_10_08, but with 3DES and Kyber (for TLS 1.2) removed, and added ML-KEM support */
+const struct s2n_security_policy security_policy_pq_tls_1_2_2024_10_08 = {
+    .minimum_protocol_version = S2N_TLS12,
+    .cipher_preferences = &cipher_preferences_20241008,
+    .kem_preferences = &kem_preferences_pq_tls_1_3_mixed_2024_10,
+    .signature_preferences = &s2n_signature_preferences_20200207,
+    .ecc_preferences = &s2n_ecc_preferences_20200310,
+};
+
+/* Same as security_policy_pq_tls_1_2_2023_04_10, but with 3DES and Kyber (for TLS 1.2) removed, and added ML-KEM support */
+const struct s2n_security_policy security_policy_pq_tls_1_2_2024_10_08_gcm = {
```

Comment:
The `security_policy_pq_tls_1_2_2023_04_10` policy only contained `x25519_kyber-512-r3` and `secp256r1_kyber-512-r3`. Is there a reason all of the other kyber preferences need to be added in this version? Or should just the ML-KEM preferences be added?